src/Bicep.RegistryModuleTool.IntegrationTests/Commands/ValidateCommandTests.cs

```diff
@@ -92,7 +93,7 @@ public void Invoke_InvalidFiles_WritesErrorsToConsole()
                 $@"The file ""{fileSystem.Path.GetFullPath(MetadataFile.FileName)}"" is invalid:
   #/summary: Value is not longer than or equal to 10 characters
 ".ReplaceLineEndings(),
-                $@"The file ""{fileSystem.Path.GetFullPath(ReadmeFile.FileName)}"" is modified or outdated. Please regenerate the file to fix it.
+                $@"The file ""{fileSystem.Path.GetFullPath(ReadmeFile.FileName)}"" is modified or outdated. Please run `brm generate` to regenerate it.
```

Comment:
Should we use `"` instead of "`"? In console, code wrapped by backticks won't be formatted as markdown code section.
```suggestion
                $@"The file ""{fileSystem.Path.GetFullPath(ReadmeFile.FileName)}"" is modified or outdated. Please run ""brm generate"" to regenerate it.
```